source/configuration_manual/event_filter.rst

```diff
@@ -92,6 +92,17 @@ A more complicated example::
   event=abc OR (event=def AND (category=imap OR category=lmtp) AND \
     NOT category=debug AND NOT (bytes_in<1024 OR bytes_out<1024))
 
+.. versionadded:: v3.0.0
+
+Sizes can now also be expressed using the unit values ``B`` - which represents
+single byte values - as well as ``K``, ``M``, ``G`` and ``T`` which are all
+powers of 1024.
```

Comment:
lowercase 'k' for kilo.

Reply:
I can change it of course, but a quick search reveals that upper case "K" is used for Base 2 (i.e. 1024 bytes), whereas lower case "k" is used for Base 10 (i.e. 1000 bytes). If I interpret the code correctly this change uses the Base 2 version, so it _should_ be upper case "K". My reference: https://en.wikipedia.org/wiki/Kilobyte

I just wanted to make sure that I understand this correctly. Also whether M/G/T need to be upper/lower case, since I would not like to switch cases in the same sentence.

Reply:
Huh, I had never heard of uppercase K intentionally used in that way. But I guess we can keep it as K then. Looks like there are somewhat longer explanation in https://github.com/ScriptFUSION/ByteFormatter/issues/5#issuecomment-168195254